### ISSUE TITLE
Mix it up

### ISSUE THREAD

**Issue by killercup** [killercup/macromusic#1]
How about we allow people to play more than one note at a time?

```rust
sick_beat!(
    hardcore_bass: (220 4, 240 2, 220 4),
    noise: (600 3, 590 1, 610 1, 600 3),
    actual_music: (…),
);
```

- The names are just for the dev.
- Repeat each section until the longest ends

**Comment by killercup** [comment]
If you ever want to spend a weekend fiddling with weird sound things to make a Hello Rust episode about macros, this is your issue, @mre!

*Update:* https://github.com/hello-rust/show/issues/23

**Comment by mre** [comment]
Yeah let's make that happen some time. You might want to collaborate on that episode. 😄
